extensions/analyticsdx-vscode-templates/src/autoInstall/completions.ts

```diff
@@ -5,31 +5,12 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { findNodeAtLocation, Location, Node as JsonNode } from 'jsonc-parser';
+import { Location, Node as JsonNode } from 'jsonc-parser';
 import * as vscode from 'vscode';
 import { TemplateDirEditing } from '../templateEditing';
 import { isValidRelpath } from '../util/utils';
 import { VariableRefCompletionItemProviderDelegate } from '../variables';
 
-function zeropad(n: number, len = 2) {
```

Comment:
I moved these and `appendDefaultValue` into the base class, to share the logic for doing code-completion in `values` in a readiness.json.